feat: Add langchain llamastack Integration example notebook

slekkala1 opened the pull request:
# What does this PR do?
The notebook was reverted as it has some local paths, I missed correcting. Trying with corrections now


## Test Plan
Ran the Jupyter notebook

mattf reviewed (commented):
ways to make this simpler -
- `!pip install` everything in one step
- use `pkill -f llama_stack.core.server.server -e`
- use `uv run --with llama-stack llama stack build ... --run` for one step install & startup
- avoid `!pip install` and `subprocess(... "pip"...)`
- the vector_store api will do this chunking for you
- pass `base_url` and `api_key` to `ChatOpenAI` instead of seting env vars



questions -
- why do you need to adjust `UV_SYSTEM_PYTHON`?
- why are so many models listed?

slekkala1 commented:
> ways to make this simpler -
> 
> * `!pip install` everything in one step
> * use `pkill -f llama_stack.core.server.server -e`
> * use `uv run --with llama-stack llama stack build ... --run` for one step install & startup
> * avoid `!pip install` and `subprocess(... "pip"...)`
> * the vector_store api will do this chunking for you
> * pass `base_url` and `api_key` to `ChatOpenAI` instead of seting env vars
> 
> questions -
> 


> * why do you need to adjust `UV_SYSTEM_PYTHON`?
For this, I have added a comment above, just in case user has this set `UV_SYSTEM_PYTHON`
> * why are so many models listed?
Removed the long list and made it short just of the provider used in the tutorials

@mattf thanks for the helpful comments! I have made it simpler with the suggestions above.
